src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -281,7 +289,13 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
     const timeRange = sceneGraph.getTimeRange(this);
 
     const timeRangeValue = timeRange.state.value;
-    const filters = sceneGraph.lookupVariable(VAR_LABELS, this)! as AdHocFiltersVariable;
+    const filters = getLabelsVariable(this);
+
+    if (!filters) {
```

Comment:
do we want to move these console.errors into the helpers?

Reply:
I've been thinking about it but I'm still unsure. There are places where we throw an Error if the variable is not found, so that means that we expect them to be defined all the time, or else the app explodes. If that is really the expectation, I would maybe move that to these functions, but we need to throw, or else we still need to return `null` and the consumer of the function needs to conditionally check for that.

Reply:
To summarize. If the variable is not found, should we: 
- Assume it should be defined and returned by the function, or else throw.
- Assume it can potentially be undefined and let the Scene/Component handle it.

Reply:
Assuming it should be defined and failing loudly is the best pattern IMO, but if we're worried about introducing runtime errors in some situation we're not currently aware of, maybe we just log the error for now, and come back and harden these up when we're confident this isn't happening in prod?

Reply:
I guess it's still in public preview, so maybe being bold is the right answer right now

Reply:
Yeah, silently failing is definitely the worst option. Let's fail loudly and see where it takes us. Thanks!

Reply:
Done! This simplified the code quite a bit, which is very nice.